resources/definitions/tronxy_xy2_pro.def.json

```diff
@@ -0,0 +1,30 @@
+{
+    "version": 2,
+    "name": "Tronxy XY2 Pro",
+    "inherits": "tronxy_base",
+    "metadata": {
+        "quality_definition": "tronxy_base",
+        "file_formats": "text/x-chitu-g-code;text/x-gcode", 
+        "visible": true
+    },
+    "overrides": {
+        "machine_name": { "default_value": "tronxy xy2 pro" },
+        "machine_width": { "default_value": 255 },
+        "machine_depth": { "default_value": 255 },
+        "machine_height": { "default_value": 250 },
```

Comment:
The website lists the height as 260mm: https://www.tronxy.com/xy-series/xy-2-pro.html

Did you make a typo here?

Reply:
Yes, it was a type. I looked in the wrong line of the datasheet